framework/src/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -204,7 +204,10 @@
      */
     protected TypeArgumentInference typeArgumentInference;
 
-    /** To cache the supported type qualifiers. */
+    /** To cache the supported type qualifiers.
+     * this field should use {@code getSupportedQuals()} to access,
```

Comment:
This comment doesn't make sense.  It should be something like "Caches the supported type qualifiers. Call {@link #getSupportedQuals} rather than using this field directly as it may not have been initialized."